src/main/kotlin/com/github/adeynack/jsoncmp/ComparisonBase.kt

```diff
@@ -0,0 +1,25 @@
+package com.github.adeynack.jsoncmp
+
+import java.io.File
+
+abstract class ComparisonBase {
```

Comment:
But then it's not possible to have `protected` stuff. That's the one thing that I miss from the Scala `trait` :(